types/priv_validator/sign_info.go

```diff
@@ -0,0 +1,238 @@
+package types
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	crypto "github.com/tendermint/go-crypto"
+	data "github.com/tendermint/go-wire/data"
+	"github.com/tendermint/tendermint/types"
+)
+
+// TODO: type ?
```

Comment:
step is exposed in LastSignedInfo, so don't think this is a good idea because of that